app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -405,12 +419,22 @@ def handle_unsupported_secondary_id(enrollment, response)
     )
   end
 
+  # MW: This could be inlined but it feels a little cleaner this way:
+  def fraud_pending?(enrollment)
+    enrollment.profile&.fraud_pending_reason.present?
+  end
+
   def process_enrollment_response(enrollment, response)
     unless response.is_a?(Hash)
       handle_response_is_not_a_hash(enrollment)
       return
     end
 
+    # We want to deactivate them regardless of status:
```

Comment:
I.e., regardless of `response['status']`. We'll always take this step, and then we just needed to tweak the success case to not fully activate them.